docs/topic-guides/kotlin/relevancy.md

```diff
@@ -0,0 +1,75 @@
+# Relevancy
+
+The `relevancy` component enables tracking the user's interests and building an interest vector based on the URLs they visit.
```

Comment:
Couple thoughts here:
  - How about something like "tracks the user's interests locally, without sharing any data over the network".  This privacy feature is a key part of the component.
  - Maybe prefix "building an interest vector", with "the component currently supports".  The goal is to add more ways to track interests in the future.